llvm-project-overlay/libcxx/BUILD.bazel

```diff
@@ -1,6 +1,115 @@
 load("@rules_ll//ll:defs.bzl", "ll_library")
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
 
+STD_PARTITIONS = [
+    "algorithm",
+    "any",
+    "array",
+    "atomic",
+    "barrier",
+    "bit",
+    "bitset",
+    "cassert",
+    "cctype",
+    "cerrno",
+    "cfenv",
+    "cfloat",
+    "charconv",
+    "chrono",
+    "cinttypes",
+    "climits",
+    "clocale",
+    "cmath",
+    "codecvt",
+    "compare",
+    "complex",
+    "concepts",
+    "condition_variable",
+    "coroutine",
+    "csetjmp",
+    "csignal",
+    "cstdarg",
+    "cstddef",
+    "cstdio",
+    "cstdlib",
+    "cstring",
+    "ctime",
+    "cuchar",
+    "cwchar",
+    "cwctype",
+    "deque",
+    "exception",
+    "execution",
+    "expected",
+    "filesystem",
+    "flat_map",
+    "flat_set",
+    "format",
+    "forward_list",
+    "fstream",
+    "functional",
+    "future",
+    "generator",
+    "initializer_list",
+    "iomanip",
+    "ios",
+    "iosfwd",
+    "iostream",
+    "istream",
+    "iterator",
+    "latch",
+    "limits",
+    "list",
+    "locale",
+    "map",
+    "mdspan",
+    # "memory",  # Bugged?
```

Comment:
FYI I've checked my libc++ patch and the export of `std::atomic` is indeed incorrect. We have not implemented atomiic smart pointers yet, that will export `std::atomic`.

I had already fixed this locally, but I want to do other fixes before uploading a new revision in Phabricator.